handbook/security.md

```diff
@@ -255,14 +255,96 @@ We configure Chrome on company-owned devices with a basic policy.
 
 **User experience impact**
 
-* Chrome needs to be restarted within 48 hours of patches being installed. The automatic restart happens after 19:00 and before 6:00 if the computer is running, and tabs are restored (except for incognito tabs).
+* Chrome needs to be restarted within 48 hours of patch installation. The automatic restart happens after 19:00 and before 6:00 if the computer is running, and tabs are restored (except for incognito tabs).
 * Ads considered intrusive are blocked.
 * uBlock Origin is enabled by default, and is 100% configurable, improving security and performance of browsing.
 * Endpoint Verification is used to make access decisions based on the security posture of the device. For example, an outdated Mac could be prevented access to Google Drive.
 
 ### Personal mobile devices
 
-The use of personal devices is allowed for some applications, as long as the iOS or Android device is kept up to date.
+The use of personal devices is allowed for some applications, as long as the iOS or Android device
+is kept up to date.
+
+## GitHub Security
+We need to host and collaborate on code as a company making open-source software! We do this using GitHub.
+
+This section covers our GitHub configuration. Like everything we do, we aim for the right level of security and productivity.
+
+We are concerned about the integrity of the code much more than by its confidentiality, because our
+code is open-source. This is why our configuration is aimed at protecting what is in the code, but we spend no
+effort preventing "leaks", since almost everything is public anyway.
+
+### Authentication
+
+Authentication is the lynchpin of security on SaaS applications such as GitHub.
```

Comment:
I don't know what SaaS means. Should we explain what this is?

Reply:
Done!